Sources/ITwinMobile/ITMOIDCAuthorizationClient.swift

```diff
@@ -116,6 +120,8 @@ open class ITMOIDCAuthorizationClient: NSObject, ITMAuthorizationClient, OIDAuth
     }
     internal static var currentAuthorizationFlow: OIDExternalUserAgentSession?
     private var loadStateActive = false
+    private let defaultScopes = "projects:read imodelaccess:read itwinjs organization profile email imodels:read realitydata:read savedviews:read savedviews:modify itwins:read openid offline_access"
+    private let defaultRedirectUri = "imodeljs://app/signin-callback"
```

Comment:
Why was this constant introduced? It seems to only be used once.

Reply:
With the much longer new scope, I felt that making defaultScopes a constant made it easier to read. To be consistent, I also made defaultRedirectUri a constant.

Reply:
Maybe `defaultIssuerUrl` should be added too then?

Reply:
That one relies on a variable, so isn't a constant value.